### ISSUE TITLE
Only 1 weaponskin at a time per type? (primary, secondary, melee & grenades)

### ISSUE THREAD

**Issue by metheyes** [Kxnrl/Store#149]
It seems like you can only have 1 weapon skin equipped at the same time in each type? We have slots 0-6 and even if you have skins/models for different weapons, for example one m4 skin and one ak47; they'll replace each other?

**Comment by Kxnrl** [comment]
no. weapon slot -> adjust  weapon slot, so u can equip ak, m4 , awp and more with slot 1

**Comment by metheyes** [comment]
> 
> 
> no. weapon slot -> adjust weapon slot, so u can equip ak, m4 , awp and more with slot 1

Just double checked, my AK47, AWP & M4 weaponskins are set to slot 1, but if I equip one, the other one(s) un-equip.

**Comment by metheyes** [comment]
Any update on this?
